esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -1412,20 +1420,16 @@ bool CWsDfuEx::DFUDeleteFiles(IEspContext &context, IEspDFUArrayActionRequest &r
     }
 
     StringBuffer returnStr;
-    StringBuffer auditStr(",FileAccess,WsDfu,DELETED,");
     IArrayOf<IEspDFUActionInfo> actionResults;
 
     StringArray superFiles, failedFiles;
 
-    auditStr.append(espProcess.get());
-    auditStr.append(',');
-    if (!isEmptyString(username))
-        auditStr.append(username).append('@');
-    context.getPeer(auditStr);
-
-    doDeleteSuperFiles(req.getLogicalFiles(), userdesc, superFiles, failedFiles, auditStr.str(),
+    VStringBuffer auditStr(",FileAccess,Esp,%s,DELETE", espProcess.get());
```

Comment:
I think elsewhere audits as "DELETED".. should be consistent.

Reply:
WE purposely changed the Actions from passed tense to active, but you're right it needs to be changed everywhere.

Reply:
Fine, main thing is to be consistent.

Reply:
Perhaps worth biting the bullet and adding a function and some enums etc. to build up a well formatted/consistent audit log.. 
but that is more work of course and more changes to everywhere else.